### DISCUSSION TITLE
Using search in start menu moves it to wrong positon when task menu is left

### DISCUSSION THREAD

**Discussion by Simsala91** [valinet/ExplorerPatcher#3267] [General]
I have the following issue:

I have moved the taskbar to the left with explorer patcher and am using Windows 11 start menu. This is mostly working fine. But when I open the start menu and click into the "Search for apps, settings and document" field or press a letter on the keyboard, the start menu gets moved to the bottom:

![Screenshot 2024-05-12 153437](https://github.com/valinet/ExplorerPatcher/assets/17149377/22f6954c-a85c-4dc5-94df-cfacf28869b7)

![Screenshot 2024-05-12 153451](https://github.com/valinet/ExplorerPatcher/assets/17149377/38b67c2d-f12c-4670-965f-043fe1fb2b10)

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1803

Please close this.
